packages/core/src/machines/index.ts

```diff
@@ -567,13 +583,26 @@ export const createAuthMachine = ({
         hasRefreshToken: (ctx) => !!ctx.refreshToken.value,
         isAutoRefreshDisabled: () => !autoRefreshToken,
         isAutoSignInDisabled: () => !autoSignIn,
-        refreshTimerShouldRefresh: (ctx) =>
-          ctx.refreshTimer.elapsed >
-          Math.max(
-            (Date.now() - ctx.accessToken.expiresAt.getTime()) / 1_000 - TOKEN_REFRESH_MARGIN,
-            refreshIntervalTime
-          ),
-
+        refreshTimerShouldRefresh: (ctx) => {
+          const expirestAt = ctx.accessToken.expiresAt
+          if (!expirestAt) return false
```

Comment:
I would suggest to use curly braces here

```ts
if (!expiresAt) {
  return false
}

// ...
```

Reply:
ha! bad habbit :)